packages/app-store/stripepayment/lib/PaymentService.ts

```diff
@@ -62,6 +63,7 @@ export class PaymentService implements IAbstractPaymentService {
         currency: this.credentials.default_currency,
         payment_method_types: ["card"],
         application_fee_amount: paymentFee,
+        ...(paymentOptionEnum.parse(paymentOption) === "HOLD" && { capture_method: "manual" }),
```

Comment:
By setting the `capture_method` of the payment intent to "manual" we can capture the funds in the future